floodlight/io/catapultApi.py

```diff
@@ -31,63 +46,63 @@ def saveResponse(response: requests.Response, endpoint: str) -> None:
     else:
         print(f"Error: {response.status_code} - {response.text}")
 
-
-def load_responses(endpoint: str) -> List[Dict]:
+def read_response(endpoint: str) -> List[Dict]:
     """
-    Loads and returns a list of JSON data responses from the specified directory.
-
-    Args:
-        endpoint (str): The directory containing the JSON files.
-
-    Returns:
-        List[Dict]: A list of JSON data responses.
-
-    Raises:
-        FileNotFoundError: If the specified directory does not exist.
+    Loads and returns a list of response_data from the specified directory.
+
+    Parameters
+    ----------
+    endpoint : str
+        The directory containing the JSON files.
+
+    Returns
+    -------
+    List[Dict]
+        A list of response_data.
+
+    Raises
+    ------
+    FileNotFoundError
+        If the specified directory does not exist.
     """
-    response_data_json_list = []
+    response_data = []
     
-    # Check if the endpoint directory exists
     if os.path.exists(endpoint):
-        # Walk through all directories and files in the given directory
         for root, dirs, files in os.walk(endpoint):
             for filename in files:
                 if filename.endswith(".json"):
                     file_path = os.path.join(root, filename)
-                    
-                    # Load the JSON file and append the data to the list
                     with open(file_path, "r") as json_file:
                         data = json.load(json_file)
-                        response_data_json_list.append(data)
+                        response_data.append(data)
     else:
         raise FileNotFoundError(f"Directory {endpoint} does not exist.")
     
-    return response_data_json_list
-
-## for testing end 
-
-# Custom Errors
-class APIRequestError(Exception):
-    """
-    Custom error class for API request failures.
-    """
-    def __init__(self, message: str):
-        super().__init__(message)
+    return response_data
 
-def _getAllplayerFormActivity_response(baseUrl: str, api_token: str, activity_id: str) -> requests.Response:
+def get_response(baseUrl: str, api_token: str, activity_id: str) -> Tuple[List[Dict], str]:
     """
     Fetches data for all players involved in a specific activity.
 
-    Args:
-        baseUrl (str): The base URL for the API requests.
-        api_token (str): The API token for authentication.
-        activity_id (str): The unique identifier for the activity to fetch players from.
-
-    Returns:
-        requests.Response: The response object containing player data.
-
-    Raises:
-        APIRequestError: If the API request fails with a status code other than 200.
+    Parameters
+    ----------
+    baseUrl : str
+        The base URL for the API requests.

```

Comment:
Can you give an example here? E.g. if I want to access the European API it is "https://connect-eu.catapultsports.com/api/v6/"